pkg/operator/k8sutil/node.go

```diff
@@ -364,3 +364,21 @@ func GenerateNodeAffinity(nodeAffinity string) (*v1.NodeAffinity, error) {
 	}
 	return newNodeAffinity, nil
 }
+
+func GetKubernetesReadyNodes(clientset kubernetes.Interface) ([]v1.Node, error) {
```

Comment:
Is this addition vestigial now?

Reply:
yes. I implemented this function in the previous solution approach..., but now is not used.  
Taking into account that it is placed in a general utility package,  maybe is ok to leave it for future use. Do you prefer to delete it?

Reply:
Deleting it seems best since we don't want to encourage others to start reimplementing k8s scheduling logic, and this seems like the foundation of that path.

Reply:
Deleted